_ont/ont-nokia-g-010g-a.md

```diff
@@ -7,19 +7,35 @@ parent: Nokia
 
 # Hardware Specifications
 
-|                |          |
-| -------------- | -------- |
-| Vendor/Brand   | Nokia    |
-| Model          | G-010G-A |
-| Chipset        | Broadcom BCM6838  |
-| Flash          |          |
-| RAM            |          |
-| System         |          |
-| 2.5GBaseT      | No       |
-| Optics         | SC/APC   |
-| IP address     |          |
-| Web Gui        |          |
-| SSH            |          |
-| Telnet         |          |
-| Serial         |          |
-| Form Factor    | ONT      |
+|                |                  |
+| -------------- | ---------------- |
+| Vendor/Brand   | Nokia            |
+| Model          | G-010G-A         |
+| Chipset        | Broadcom BCM6838 |
+| Flash          |                  |
+| RAM            |                  |
+| System         |                  |
+| 2.5GBaseT      | No               |
+| Optics         | SC/APC           |
+| IP address     |                  |
+| Web Gui        | ✅               |
+| SSH            |                  |
+| Telnet         |                  |
+| Serial         |                  |
+| Form Factor    | ONT              |
+
+
+{% include image.html file="g-010g-a.jpg"  alt="G-010G-A" caption="Nokia G-010G-A" %}
+
+# Curiosity
+
+In some territories in the USA, in the ex BellSouth network uses ONTs labelled 'ALT Optics' that use a different length, 1310 nm for upstream and 1550 nm for downstream instead of 1490 nm.
```

Comment:
```suggestion
In some territories in the USA the ex BellSouth network uses ONTs labelled 'ALT Optics' that use a different wave length, 1310 nm for upstream and 1550 nm for downstream (instead of 1490 nm).
```